src/libAtomVM/CMakeLists.txt

```diff
@@ -150,6 +150,38 @@ else()
     endif()
 endif()
 
+include(CheckSymbolExists)
```

Comment:
maybe we should make this a CMake module or something like that, for the sake of clarity.